website/client/js/services/petServices.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+(function(){
+  angular
+    .module('habitrpg')
+    .factory('Pets', petsFactory);
+
+  petsFactory.$inject = [
+    'Content'
+  ];
+
+  function petsFactory(Content) {
+    function isBasicAnimal(name, type) {
```

Comment:
How so?
On Fri, Jul 22, 2016 at 2:12 PM Kaitlin Hipkin <notifications@github.com>
wrote:

> In website/client/js/services/petServices.js
> <https://github.com/HabitRPG/habitrpg/pull/7768#discussion_r71930528>:
>
> > @@ -0,0 +1,39 @@
> > +'use strict';
> > +
> > +(function(){
> > +  angular
> > +    .module('habitrpg')
> > +    .factory('Pets', petsFactory);
> > +
> > +  petsFactory.$inject = [
> > +    'Content'
> > +  ];
> > +
> > +  function petsFactory(Content) {
> > +    function isBasicAnimal(name, type) {
>
> This function name doesn't really make sense any more.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7768/files/65ffc1ec10986f7856e586150381153d7e7af265..8fddc1b86d4ea27b6797b4d7818aaca8c1a1eafb#r71930528>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALz_8zRGfyxercKnyXkq6PcXcyV6jS32ks5qYRYAgaJpZM4JIjd_>
> .
>